doc/examples/transform/plot_ring_detector.py

```diff
@@ -0,0 +1,132 @@
+"""
+=============================
+Ridge Directed Ring Detection
+=============================
+
+Robust and highly performant ring detection algorithm for 3d particle tracking
```

Comment:
```suggestion
Robust and highly performant ring detection algorithm for 3D particle tracking
```
from my impression on the convention used in scikit-image's docs